components/form/Label.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { useFormContext } from 'react-hook-form';
 import classNames from 'classnames';
 
-export function Label({ fontSize, floatLabel, htmlFor, name, text }) {
+export function Label({ fontSize, floatLabel, htmlFor, labelClassName, name, text }) {
```

Comment:
Inside the Label component, we would use `className` not `labelClassName`. `labelClassName` results in tautology since we are already in the Label component.

```suggestion
export function Label({ fontSize, floatLabel, htmlFor, className, name, text }) {
```